pkg/eventing/v1alpha1/client_test.go

```diff
@@ -142,22 +142,10 @@ func TestTriggerBuilder(t *testing.T) {
 		assert.DeepEqual(t, expected, b.Build().Spec.Filter)
 	})
 
-	t.Run("update filter with only deletions", func(t *testing.T) {
+	t.Run("update filter with add and update", func(t *testing.T) {
```

Comment:
this description reads odd... any way to make this clearer?

Reply:
fixed. Thank you.